tests/engine/gan_system_test.py

```diff
@@ -0,0 +1,81 @@
+import torch
+from torch import nn, optim
+from torch.utils import data
+from pytorch_lightning import Trainer
+from torch.optim.lr_scheduler import ReduceLROnPlateau
+from asteroid.engine.gan_system import GanSystem
+from torch.nn.modules.loss import _Loss
+from asteroid.losses import PITLossWrapper, pairwise_neg_sisdr
+
+
+class DummyDataset(data.Dataset):
+    def __init__(self):
+        self.inp_dim = 10
+        self.out_dim = 10
+
+    def __len__(self):
+        return 20
+
+    def __getitem__(self, idx):
+        return torch.randn(1, self.inp_dim), torch.randn(1, self.out_dim)
+
+
+class GeneratorLoss(_Loss):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, estimates, targets, est_labels):
+        loss = torch.mean((est_labels - torch.ones_like(est_labels)) ** 2)
+        return loss
+
+
+class DiscriminatorLoss(_Loss):
+    def __init__(self):
+        super().__init__()
```

Comment:
Those lines are not needed. 
If you don't subclass a method, it stays the same. 
Same above